src/components/InlineFilter/index.tsx

```diff
@@ -0,0 +1,21 @@
+import type React from 'react';
+import FilterChildren from '../FilterChildren';
+
+type InlineFilterProps = {
+  children: JSX.Element
```

Comment:
```suggestion
  children: React.ReactNode
```

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d076add9f29db350a19bd94c37b197729cc02f87/types/react/index.d.ts#L822